vessel-runtime/build.gradle.kts

```diff
@@ -7,22 +7,18 @@ plugins {
 }
 
 android {
-    compileSdkVersion(30)
-    buildToolsVersion = "30.0.2"
+    compileSdk = 31
```

Comment:
make sure the changelog reflects this change

Reply:
Will do!